specs/plugin.spec.ts

```diff
@@ -26,16 +26,12 @@ describe('The "gulp-svg2png" plugin', () => {
 		const image = Helper.createTestFile();
 
 		stream.on('data', (png: any) => {
-			expect(png.path).to.equal('./specs/assets/twitter.png');
+			expect(png.path).to.equal('specs/assets/twitter.png');
```

Comment:
`vinyl` automatically strips `./` - that's why it was failing. 

![image](https://user-images.githubusercontent.com/2566530/52245570-2add1d80-28b0-11e9-9c05-89347eb43233.png)